nf_core/create.py

```diff
@@ -3,6 +3,7 @@
 organization's specification based on a template.
 """
 from genericpath import exists
+from pathlib import Path
```

Comment:
I generally try to avoid this kind of thing, as I find direct usage of `Path` in the code out of no-where really difficult to interpret, `pathlib.Path` is much clearer imho. So unless used a lot I prefer to just `import pathlib`

Reply:
true, I have a tendency to do that - but you're absolutely right 👍